### ISSUE TITLE
Crash while using the Speen tool

### ISSUE THREAD

**Issue by nutti** [nutti/Screencast-Keys#4]
**System Information**

* Blender version: 2.8
* Add-on version: 3.0

**Description about the bug**  
Crash while using the Speen tool

**Comment by javimixet** [comment]
Blender 2.8rc crash when using Spin tool:
```
WARN (bpy.rna): /home/sources/blender-release/source/blender/python/intern/bpy_rna.c:1457 pyrna_enum_to_py: current value '20517' matches no enum in 'Event', '(null)', 'type'
Traceback (most recent call last):
  File "/home/ii/.config/blender/2.80/scripts/addons/screencastkeys/ops.py", line 587, in modal
    event_type = EventType[event.type]
  File "/home/ii/Downloads/blender-2.80rc1-linux-glibc217-x86_64/2.80/python/lib/python3.7/enum.py", line 349, in __getitem__
    return cls._member_map_[name]
KeyError: ''

location: <unknown location>:-1

location: <unknown location>:-1
Writing: /tmp/blender.crash.txt
```

**Comment by nutti** [comment]
This is now fixed by #10
